hugging_py_face/computer_vision.py

```diff
@@ -45,4 +48,21 @@ def object_detection(self, inputs: Union[Text, List], model: Optional[Text] = No
         if type(inputs) == list:
             return self._query(inputs, model=model, task="object-detection")
         elif type(inputs) == str:
-            return self._query(inputs, model=model, task="object-detection")
\ No newline at end of file
+            return self._query(inputs, model=model, task="object-detection")
+        
+    def object_detection_in_df(self, df: DataFrame, column: Text, model: Optional[Text] = None) -> DataFrame:
```

Comment:
This is really awesome! I was just thinking that since this is object detection, we might need to include the coordinates as well? What do you think? We can maybe include these in separate columns in the resulting DataFrame?

Reply:
All thanks goes to you good sir! I'm just a fan who hopes I'm not making things worse. Coordinates will definitely be helpful especially if we want to augment images to pinpoint objects. I was messing around in my development area but need a little more time to try that without adding too much bloat to the codebase.

Reply:
You are making it much better! I was wondering how to tackle the object detection task and your help is much appreciated. Please take your time and reach out to me anytime if you want to talk it out.